src/chrome/content/rules/DENX.de.xml

```diff
@@ -1,13 +1,34 @@
-<ruleset name="DENX.de" platform="mixedcontent">
+<!--
+	Different HTTP/HTTPS content:
+		- ftp.denx.de
+		- ns.denx.de
+		- theia.denx.de
 
-	<target host="denx.de" />
-	<target host="www.denx.de" />
+	Invalid certificate:
+		- ns2.denx.de
+		- phoibe.denx.de
 
+	Login required:
+		- gitlab.denx.de
+		- owncloud.denx.de
 
-	<securecookie host="^www\.denx\.de$" name=".+" />
+	MCB:
+		- theia.denx.de
 
+	Passive mixed content:
+		- denx.de
+		- www.denx.de
+		- theia.denx.de
 
-	<rule from="^http:"
-		to="https:" />
+	Service unavailable:
+		- jabber.denx.de
+-->
+<ruleset name="DENX.de" platform="mixedcontent">
```

Comment:
Is there any active mixed content? Otherwise the `mixedcontent` can be removed.

Reply:
There are two CSS files being loaded via HTTP on the `theia` subdomain, but since it's not listed as target anymore, `mixedcontent` has been removed.

Reply:
Thanks!